js/main.js

```diff
@@ -0,0 +1,63 @@
+var random = [];
+
+// Feature 1 START
+
+var $xhrRandom = new XMLHttpRequest();
+var randomPage = Math.floor(Math.random() * 934 + 1);
+$xhrRandom.open('GET', 'https://api.jikan.moe/v4/anime?sfw=true&page=' + randomPage);
+$xhrRandom.responseType = 'json';
+$xhrRandom.addEventListener('load', loading);
+
+function loading(event) {
+  random.push($xhrRandom.response);
+  var randomArray = [];
+  for (var a = 0; a < random[0].data.length; a++) {
+    randomArray.push(random[0].data[a]);
+  }
```

Comment:
I don't understand the purpose of `random` and `randomArray`. You should be able to just choose three objects at random from `$xhrRandom.response.data` and use those objects to create your DOM trees.